driver/controller.go

```diff
@@ -33,10 +33,14 @@ import (
 )
 
 const (
-	_  = iota
+	_ = iota
+	// KB is a Kibibyte https://en.wikipedia.org/wiki/Kibibyte.
 	KB = 1 << (10 * iota)
+	// MB is a Mebibyte https://en.wikipedia.org/wiki/Mebibyte.
 	MB
+	// GB is a Gibibyte https://en.wikipedia.org/wiki/Gibibyte.
 	GB
+	// TB is a Tebibyte https://en.wikipedia.org/wiki/Tebibyte.
 	TB
```

Comment:
These are not needed and makes it hard to read. Can you please remove these comments?

Reply:
@fatih I can definitely remove those comments, however, `golint` will still complain about those since they're exported. They appear in the GoReportCard as `Line 37: warning: exported const KB should have comment (or a comment on this block) or be unexported (golint)` and so on for the rest. https://goreportcard.com/report/github.com/digitalocean/csi-digitalocean#golint

The goal of this cosmetic PR is just to achieve 100% on all GoReportCard categories except `gocyclo`.

Reply:
All commits squashed into a single commit.